great_expectations/util.py

```diff
@@ -1899,6 +1905,37 @@ def get_context(  # noqa: PLR0913
     """
     project_config = _prepare_project_config(project_config)
 
+    if mode is not None:
+        if mode == "ephemeral":
+            return _get_ephemeral_context(
+                project_config=project_config,
+                runtime_environment=runtime_environment,
+            )
+        elif mode == "file":
+            return _get_file_context(
+                project_config=project_config,
+                context_root_dir=context_root_dir,
+                runtime_environment=runtime_environment,
+            )
+        elif mode == "cloud":
+            return _get_cloud_context(
+                project_config=project_config,
+                context_root_dir=context_root_dir,
+                runtime_environment=runtime_environment,
+                cloud_base_url=cloud_base_url,
+                cloud_access_token=cloud_access_token,
+                cloud_organization_id=cloud_organization_id,
+                cloud_mode=cloud_mode,
+                ge_cloud_base_url=ge_cloud_base_url,
+                ge_cloud_access_token=ge_cloud_access_token,
+                ge_cloud_organization_id=ge_cloud_organization_id,
+                ge_cloud_mode=ge_cloud_mode,
+            )
+        else:
+            raise ValueError(
+                f"Unknown mode {mode}. Please choose one of: ephemeral, file, cloud."
+            )
```

Comment:
As is, this is a breaking change for users - if we want to make this required, we should warn before making it mandatory

Reply:
jk didn't realize `if mode is not None:`